### PR TITLE
Godforge finale

### PR DESCRIPTION
**Pull request by GDCloudstrike**
Hopefully final godforge PR. 

https://github.com/user-attachments/assets/aaae1efb-b74b-4419-93d5-cdb52bbabf8a

Doc: https://docs.google.com/document/d/1Lpt73uBFvdEvF_Wx0dfHodfxW4zCSyxlcRwd9XUhH-A/edit?usp=sharing

### CODE REVIEW

File: src/main/resources/assets/tectech/lang/en_US.lang [hunk 1]
```diff
@@ -1094,7 +1094,7 @@ gt.blockmachines.multimachine.FOG.modules=Modules
...
...rements. In general, each upgrade can only be unlocked if the prior one is unlocked and there are enough available graviton shards. One exception to this is the first upgrade, as that one has no prior upgrade. Some upgrades can also have extra material costs, which are denoted next to the unlock button if applicable. If an upgrade has more than 1 connected prior upgrade, then there are two options, either the upgrade requires ALL connected prior upgrades or AT LEAST ONE (indicated by the connection's color). Upgrades can be refunded by simply pressing the unlock button again, but this only works if ALL connected later upgrades are not active/unlocked. The block of upgrades following the unlock of the Heliufusion Exoticizer module are special, as they are what is considered §Bthe Split§6. As the name suggests, only one path out of the three may be chosen at first, and the others will be locked. Each path has specialized buffs that are primarily targeted towards a specific module, and thus affect other module types with reduced effect (unless stated otherwise). The amount of unlockable paths depends on the amount of rings the Godforge has, which in turn are limited by later upgrades.
```
Comment (Connor-Colenso) [inline src/main/resources/assets/tectech/lang/en_US.lang:1098]: Upgrades*

**Comment by Caedis** [comment]
Make sure you reenable the multi outside of dev envs

**Comment by Dream-Master** [comment]
@GDCloudstrike what is the status of this pr?

**Comment by GDCloudstrike** [comment]
> @GDCloudstrike what is the status of this pr?

Should be mergeable by the weekend

File: src/main/java/tectech/loader/recipe/Godforge.java [hunk 2]
```diff
@@ -124,143 +153,219 @@ public void run() {
...
-                ItemStack[] solids_t1_1step = { Materials.Lead.getDust(1), Materials.Plutonium241.getDust(1),
-                    Materials.Thorium.getDust(1), Materials.Naquadria.getDust(1) };
-                FluidStack[] solid_plasmas_t1_1step = { Materials.Lead.getPlasma(144),
-                    Materials.Plutonium241.getPlasma(144), Materials.Thorium.getPlasma(144),
-                    Materials.Naquadria.getPlasma(144) };
+        }
+        // Fusion tier 4-5
+        {
+            // Single step
+            ItemStack[] solids_t1_1step = { Materials.Lead.getDust(1), Materials.Plutonium241.getDust(1),
```
Comment (NotAPenguin0) [inline src/main/java/tectech/loader/recipe/Godforge.java:230]: Naquadria plasma has disappeared

File: src/main/java/tectech/loader/recipe/Godforge.java [hunk 3]
```diff
@@ -675,4 +745,78 @@ public void run() {
...
+
+            fluidOutputs.addAll(Arrays.asList(recipe.mFluidOutputs));
+
+            GTRecipeBuilder builder = GTValues.RA.stdBuilder()
+                .noOptimize()
+                .itemOutputs(itemOutputs.toArray(new ItemStack[0]))
+                .fluidOutputs(fluidOutputs.toArray(new FluidStack[0]))
+                .duration(recipe.mDuration)
+                .eut(recipe.mEUt)
+                .specialValue(recipe.mSpecialValue);
```
Comment (NotAPenguin0) [inline src/main/java/tectech/loader/recipe/Godforge.java:783]: Unless coil heat is irrelevant for the molten module, this should probably also copy over the `COIL_HEAT` metadata tag.

**Review by NotAPenguin0 — CHANGES_REQUESTED** [review]
Almost good to go

**Review by NotAPenguin0 — APPROVED** [review]

**Pull request merged**